prinz_pmml/src/main/scala/pl/touk/nussknacker/prinz/pmml/model/VectorMultimapUtils.scala

```diff
@@ -0,0 +1,19 @@
+package pl.touk.nussknacker.prinz.pmml.model
+
+import pl.touk.nussknacker.prinz.util.collection.immutable.VectorMultimap
+
+object VectorMultimapUtils {
+  implicit class VectorMultimapAsRowset[K, V](val inputMap: VectorMultimap[K, V]) {
+    def forEachRow[T](f: Map[K, V] => T) : IndexedSeq[T] = {
+      val totalTuples = inputMap.map(_._2.length).max
+      val iteratorMap = inputMap.mapVectors(_.iterator)
```

Comment:
name "iteratorsMap" makes it more clear

Reply:
Changed both "iterator" and "input" to keep things consistent.